aspnet/security/anti-request-forgery.md

```diff
@@ -0,0 +1,85 @@
+
+  ## Fundamentals  ### Cookie based authentication
+
+Once a user has authenticated using their username and password, they are issued a token that can be used to identify them and validate that they have been authenticated. The token is stored as a cookie that accompanies every request the client makes. Generating and validating this cookie is done by the OWIN Cookie Authentication middleware. ASP.NET Core provides cookie [middleware](../fundamentals/middleware.md#fundamentals-middleware.md) which serializes a user principal into an encrypted cookie and then, on subsequent requests, validates the cookie, recreates the principal and assigns it to the `User` property on `HttpContext`.
+
+When cookie  is used, The authentication cookie is just a container for the forms authentication ticket. The ticket is passed as the value of the forms authentication cookie with each request and is used by forms authentication, on the server, to identify an authenticated user.
+
+When a user is logged in to a system, a user session is created on the server side and it is stored in the database or in some kind of in-memory storage systems. The system generates a session key that points to the actual session in the database and it is attached to a client side cookie. After that every time when a user asks for a page which needs authorization, the session key in the cookie is finds the appropriate user session in the database. Then the system checks if the user from whom the user session is stored has the privilege to access that page. If it does, then eventually the application takes the user to that page. As you can see we are using cookies to make the application stateful. USERTOKENS Token based authentication on the other hand doesn’t store any kind of session in the database. Instead when a user is logged in he is issued with a token (not antiforgery token). This token is self-contained. It contains all the data that is required to validate the token as well
+as user information through claims. When a user wants to access some authorized pages the token is send to the server with an additional authorization header in form of Bearer {token} (Recall from previous web api applications with single user authentication enabled). This makes the application stateless since in every subsequent request the token is passed in the request for server side validation. One thing to remember is this token is not encrypted rather it is encoded. On the server side the token can be decoded to have the raw information about the token. To send the token in subsequent requests, you can either store it in browser’s local storage or in a cookie. You don’t have to worry about anything related to XSRF if your token is stored in the local storage. But you have to deal with XSRF if you store it in a cookie.
+
```

Comment:
I see the issue - you're looking in /aspnet/anti-request-forgery when the doc in question is actually in /security/anti-request-forgery.md. I didn't touch the one in /aspnet and I'm not sure what we should do with it (other than update it, separately, to reflect the CSRF capabilities of ASP.NET (non-core)).